community/DOCS-CONTRIBUTING.md

```diff
@@ -98,3 +103,101 @@ Code samples follow a similar strategy, where most code samples should be
 located in the `docs` repository. If there are code samples used for testing
 that are only expected to be used by contributors, those samples are put in the
 `samples` folder within the component repo.
+
+## Docs contributor roles
+
+Because contributing to the documentation requires a different skill set than
+contributing to the Knative code base, we've defined the roles of
+documentation contributors seperately from the roles of code contributors.
+
+If you're looking for code contributor roles, see [ROLES](./ROLES.md).
+
+### Member
+
+Established contributor to the Knative docs.
+
+Members are continuously active contributors in the community. They can have
+issues and PRs assigned to them, may participate in working group meetings, and
+pre-submit tests are automatically run for their PRs. Members are expected to
+remain active contributors to the Knative documentation.
+
+All members are encouraged to help with PR reviews, although each PR
+must be reviewed by an official [Approver](#approver). In their review,
+Members should be looking for correctness of the documentation, adherence to the style guide,
+good spelling and grammar (writing quality), intuitive organization, and strong documentation usability.
+Members should be proficient in at least one of these review areas.
+
+### Requirements
+
+- Has made multiple contributions to the project or community. Contributions may
+  include, but are not limited to:
+
+  - Authoring or reviewing PRs on GitHub in Knative/Docs or Knative/Website
+
+  - Filing or commenting on issues on GitHub
+
+  - Contributing to working group or community discussions
+
+- Subscribed to
+  [knative-dev@googlegroups.com](https://groups.google.com/forum/#!forum/knative-dev)
+
+- Actively contributing to 1 or more areas.
+
+- Sponsored by 1 approver.
```

Comment:
Does this mean that the one who want to be a Member need to contact one approver directly if s/he think s/he is qualify enough? ;-)